neuronsimulator/views.py

```diff
@@ -1,5 +1,42 @@
 from django.shortcuts import render
+from neuronsimulator.forms import ParamForm
+from neuronsimulator.models import Neuron
 
 
 def home(request):
     return render(request, "home.html")
+
+
+def list_input_params(request):
+
+    context = {}
+    params = {}
+
+    # all neurons
+    neurons = Neuron.objects.all()
+
+    # form
+    my_form = ParamForm()
+    if request.method == "POST":
+        my_form = ParamForm(request.POST)
+        # print(request.POST)
+        if my_form.is_valid():
+            stim_neuron = my_form.cleaned_data["stim_neuron"]
+            resp_neurons = my_form.cleaned_data["resp_neurons"]
+            nt = my_form.cleaned_data["nt"]
+            dt = my_form.cleaned_data["dt"]
+            stim_type = my_form.cleaned_data["stim_type"]
+            dur = my_form.cleaned_data["dur"]
+
+            # TODO: will add code for validations
```

Comment:
C. Just a note that the validation code doesn't necessarily go here.  You can override an `is_valid` method in the form class (if I recall correctly).  Those methods are checked automatically.  And if this was a class, there would be a pair of methods here that say what to do if the form submission was (or wasn't) valid.  I'm not sure how that works in a function though.  There are also widgets for form validation that happens before form submission that are handled by form management.  You can insert labels for each field that show what errors to display for which fields where and how you want them to appear (though I haven't quite gotten the hang of the ins and outs of that yet).